### ISSUE TITLE
Tabs component give axe accessibility violation 

### ISSUE THREAD

**Issue by MichaelJohnBradley** [react-bootstrap/react-bootstrap#5530]
## Describe the bug

When using the tabs component and then running the axe accessibility extension tool in chrome there is a violation reported stating that "ARIA role must be appropriate for the element". The suggestion to fix this is "ARIA role tablist is not allowed for the given element". Upon inspection of the element the issue is with the `<nav class="nav nav-tabs" role="tablist">` . This is to conform to "best practice".

## To Reproduce

Create a react component and import the tabs and tab component from react-bootstrap.

`import React from 'react';`
`import { Tabs, Tab } from 'react-bootstrap';`
`const ExampleTabs = () => {`
 ` return (`
      `<Tabs defaultActiveKey='tab1'>`
        `<Tab eventKey='tab1' title='Tab1'>`
          `Tab 1 contents`
        `</Tab>`
        `<Tab eventKey='tab2' title='Tab2'>`
          `Tab 2 contents`
        `</Tab>`
      `</Tabs>`
  `);`
`};`

export default ExampleTabs;`

##Code sandbox example 
https://codesandbox.io/s/solitary-wind-3nrt8

## Expected behavior

I would expect that there are no accessibility violations if the Tabs component is fully compliant with the standards

## Screenshots

![image](https://user-images.githubusercontent.com/28536931/99517381-361b1380-2987-11eb-9075-fcd48a0eb1a7.png)
![image](https://user-images.githubusercontent.com/28536931/99517418-416e3f00-2987-11eb-9800-5ae45806dd6d.png)

## Environment (please complete the following information)

- Operating System: Windows 10
- Browser, Version Version 86.0.4240.111
- React-Bootstrap Version 1.3.0

**Comment by kyletsang** [comment]
Bootstrap is flexible with the tab markup, but any change we make would be breaking.

@jquense thoughts?

Ref upstream docs regarding navs:
> Note that if you’re using `<nav>`, you shouldn’t add `role="tablist"` directly to it, as this would override the element’s native role as a navigation landmark.

Ref: https://getbootstrap.com/docs/4.5/components/navs/#javascript-behavior

**Comment by kyletsang** [comment]
Fixed in v2 alpha
